docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3'
+services:
+  jenkins:
+    image: jenkins/jenkins:lts
+    ports:
+      - 8080:8080 #expose port 8080 to host
+    volumes:
+      - JENKINS_DATA:/var/jenkins_home #mount volume to persist data
+    networks:
```

Comment:
Any specific reason why we are not using the default docker compose network ?

Reply:
We can use the default compose network but with a custom network, the user using this compose file will have the ability to configure additional options, also it will be flexible to use with other compose files.